VAMobile/src/components/Templates/CategoryLanding.tsx

```diff
@@ -37,6 +39,7 @@ export const CategoryLanding: FC<CategoryLandingProps> = ({ title, headerButton,
   const theme = useTheme()
   const navigateTo = useRouteNavigation()
   const screenReaderEnabled = useIsScreenReaderEnabled(true)
+  const waygateScreen = 'WG_' + useNavigationState((state) => state.routes[state.routes.length - 1]?.name)
```

Comment:
This isn't a big deal since it's not doing much, but since this is used in multiple templates, it could be moved into its own utility hook

Reply:
I'm on board with this and will get it in the next iteration of changes. Want to get this one passed asap.